.pre-commit-config.yaml

```diff
@@ -20,6 +20,7 @@ repos:
         files: ^(telegram|examples)/.*\.py$
         args:
           - --rcfile=setup.cfg
+          - --jobs=0
```

Comment:
Would you mind adding a comment explaining what this does and/or linking to the pylint docs?